src/main/java/org/jenkinsci/plugins/workflow/flow/FlowExecutionList.java

```diff
@@ -5,6 +5,7 @@
 import com.google.common.util.concurrent.FutureCallback;
 import com.google.common.util.concurrent.Futures;
 import com.google.common.util.concurrent.ListenableFuture;
+import com.google.common.util.concurrent.MoreExecutors;
```

Comment:
As in https://github.com/jenkinsci/workflow-support-plugin/pull/114#discussion_r585678752 can this be made to tolerate either old or new Guava in the short term?